willa/tind/fetch.py

```diff
@@ -62,3 +68,86 @@ def fetch_file(file_url: str, output_dir: str = '') -> str:
         raise RecordNotFoundError('Referenced file could not be downloaded.')
 
     return saved_to
+
+def fetch_file_metadata(record: str) -> Record:
+    """Fetch file metadata for a given Tind record.
+    :raises AuthorizationError: When the TIND API key is invalid.
+    :raises Exception: for any response other then 200. 
+    :returns a json list of metadata for a given Tind record
+    """ 
+
+    status, files = tind_get(f"record/{record}/files")
+
+    if status != 200 and status < 500:
+        j = json.loads(files)
+        reason = j['reason']
+        raise TINDError(f"Status: {status} Message: {reason}.")
+    
+    return json.loads(files)
+
+def fetch_ids_search(search: str):
```

Comment:
There's no docstring for this method.  Also, methods should have two newlines between them.

```suggestion

def fetch_ids_search(search: str):
```